backend/fms_core/migrations/0049_v4_0_0.py

```diff
@@ -0,0 +1,348 @@
+        ("hs37d5", None, None, None, 9606, 3101788170),
+        ("GRCm39", "mm39", "GCA_000001635.9", "GCF_000001635.27", 10090, 2728222451),
+        ("GRCm38.p6", "mm10", "GCA_000001635.8", "GCF_000001635.26", 10090, 2730855475),
+        ("ASM985889v3", None, "GCA_009858895.3", "GCF_009858895.2", 2697049, 29903),
+        ("ASM886v2", None, "GCA_000008865.2", "GCF_000008865.2", 562, 5594605),
+        ("ASM584v2", None, "GCA_000005845.2", "GCF_000005845.2", 562, 4641652),
+        ("ROS_Cfam_1.0", None, "GCA_014441545.1", "GCF_014441545.1", 9615, 2396858295)
+    ]
+
+    with reversion.create_revision(manage_manually=True):
+        admin_user = User.objects.get(username=ADMIN_USERNAME)
+        admin_user_id = admin_user.id
+
+        reversion.set_comment(f"Add commonly used Reference Genomes.")
+        reversion.set_user(admin_user)
+
+        for assembly_name, synonym, genbank_id, refseq_id, taxon_ncbi_id, size in REFERENCE_GENOMES:
+            taxon = Taxon.objects.get(ncbi_id=taxon_ncbi_id)
+
+            reference_genome = ReferenceGenome.objects.create(assembly_name=assembly_name,
+                                                              synonym=synonym,
+                                                              genbank_id=genbank_id,
+                                                              refseq_id=refseq_id,
+                                                              taxon=taxon,
+                                                              size=size,
+                                                              created_by_id=admin_user_id,
+                                                              updated_by_id=admin_user_id)
+            reversion.add_to_revision(reference_genome)
+
+def initialize_workflows(apps, schema_editor):
+    Step = apps.get_model("fms_core", "Step")
+    Protocol = apps.get_model("fms_core", "Protocol")
+    StepSpecification = apps.get_model("fms_core", "StepSpecification")
+    Workflow = apps.get_model("fms_core", "Workflow")
+    StepOrder = apps.get_model("fms_core", "StepOrder")
+
+    STEPS = [
+        # {name, protocol_name, specifications}
+        {"name": "Extraction (DNA)", "protocol_name": "Extraction",
+         "specifications": [{"display_name": "Extraction Type", "sheet_name": "ExtractionTemplate", "column_name": "Extraction Type", "value": "DNA"}]},
+        {"name": "Extraction (RNA)", "protocol_name": "Extraction",
+         "specifications": [{"display_name": "Extraction Type", "sheet_name": "ExtractionTemplate", "column_name": "Extraction Type", "value": "RNA"}]},
+        {"name": "Sample QC", "protocol_name": "Sample Quality Control",
+         "specifications": []},
+        {"name": "Normalization (Sample)", "protocol_name": "Normalization",
+         "specifications": [{"display_name": "Normalization Type", "sheet_name": "Normalization", "column_name": "Robot Norm Choice", "value": "Sample Janus"}]},
```

Comment:
We are not supporting Sample Biomek?

Reply:
I don't want to have a workflow dedicated to a robot in a normalization step. The idea here was to prefill with a valid value (either Janus or Biomek) i think the default is Janus for Ariane so i used this one. Ideally we would have just value Sample and we could move the janus and biomek to another template column but i'm not sure it's desirable for the lab so i did this little hack.